apps/rpjs-site/components/sections/speakers.tsx

```diff
@@ -31,6 +31,13 @@ const speakers: Array<Speaker> = [
     title: 'Fodao e fodinha',
     media: 'https://github.com/DouglasSerafim',
   },
+  {
+    name: 'Ítalo Covas',
+    role: 'Frontend Developer',
+    image: 'https://github.com/ItaloCovas.png',
+    title: 'Desenvolvedor Frontend na Framework Digital',
+    media: 'https://github.com/ItaloCovas',
+  },
```

Comment:
Please remove it, here will be the speakers, soon I will recreate our organizers section